### PR TITLE
Upgrade to appwrite v1

### PR DESCRIPTION
**Pull request by fballan93**

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ environment:
   flutter: ">=2.0.0"
 
 dependencies:
-  appwrite: ^7.0.0
+  appwrite: ^8.0.0
```
Comment (lohanidamodar) [inline pubspec.yaml]: Please use the latest version `8.1.0`

**Review by lohanidamodar — CHANGES_REQUESTED** [review]
Let's also update this package version so that I can release.

File: pubspec.yaml (hunk 1 above)
Comment (fballan93) [reply pubspec.yaml]: done

**Comment by fballan93** [comment]
Any thing stopping us from merging ?

**Pull request merged**